nilearn/plotting/glass_brain.py

```diff
@@ -96,26 +96,26 @@ def _get_json_and_transform(direction):
 
     dirname = os.path.dirname(os.path.abspath(__file__))
     dirname = os.path.join(dirname, 'glass_brain_files')
-    direction_to_filename = {
-        direction: os.path.join(
+    direction_to_filename = dict([
+        (_direction, os.path.join(
```

Comment:
Yup!  And nice catch; I should have added a comment to the PR to point out that change...